implementations/rust/ockam/ockam_command/src/docs.rs

```diff
@@ -35,45 +35,60 @@ static THEME: Lazy<Option<Theme>> = Lazy::new(|| {
     Some(theme)
 });
 
+fn is_markdown() -> bool {
+    match std::env::var("OCKAM_HELP_RENDER_MARKDOWN") {
+        Ok(v) => v.eq_ignore_ascii_case("true") || v.eq_ignore_ascii_case("1"),
+        Err(_e) => false,
+    }
+}
+
+pub(crate) fn hide() -> bool {
+    match std::env::var("OCKAM_HELP_SHOW_HIDDEN") {
+        Ok(v) => !v.eq_ignore_ascii_case("true") || !v.eq_ignore_ascii_case("1"),
+        Err(_e) => true,
+    }
+}
+
 pub(crate) fn about(body: &str) -> &'static str {
-    Box::leak(highlight_syntax(body.to_string()).into_boxed_str())
+    if is_markdown() {
+        Box::leak(body.to_string().into_boxed_str())
+    } else {
+        let syntax_highlighted = highlight_syntax(body.to_string());
+        Box::leak(syntax_highlighted.into_boxed_str())
+    }
 }
 
 #[allow(unused)]
 pub(crate) fn before_help(body: &str) -> &'static str {
-    Box::leak(highlight_syntax(body.to_string()).into_boxed_str())
+    if is_markdown() {
```

Comment:
We should refactor that code with L.53, and L. 71 so that we don't have to introduce more duplication or forget about the `is_markdown` case when we introduce a new section.